pkgs/tools/security/keysmith/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
```

https://nixos.org/nixpkgs/manual/#sec-language-qt
```suggestion
{ lib
, mkDerivation
```